v4/source/system.yaml

```diff
@@ -304,3 +304,22 @@
           $ref: '#/responses/Unauthorized'
         '501':
           $ref: '#/responses/NotImplemented'
+
+  '/system/health':
+    get:
+      tags:
+        - system
+      summary: Retrieve if the system are health or unhealthy
+      description: |
+        Retrieve if the system are health or unhealthy based on the configuration `GoRoutineHealthThreshold`.
+        ##### Permissions
```

Comment:
And this to:

```
Check if the system is health or unhealthy based on the configuration setting `GoRoutineHealthThreshold`.

__Minimum server version__: 3.10
```

(Need to add minimum server version for everything past 3.9)